scripts/update_lit_checks.py

```diff
@@ -146,6 +146,9 @@ def get_command_output(args, test, lines, tmp):
         if len(commands) > 1 and commands[1].startswith('filecheck '):
             filecheck_cmd = commands[1]
             commands = commands[:1]
+        if len(commands) > 1 and commands[1].startswith('FileCheck '):
+            warn('`FileCheck` is not a known command. '
+                 'Did you mean to use `filecheck` instead?')
 
```

Comment:
could these be a chain if if-elif ending in an error? Or is the set of commands we need too long to be written out?

Reply:
The set of commands is unbounded. Every command except for `filecheck` is passed through directly to the underlying system. So the only "known command" is `filecheck`, but there are no invalid commands.

Reply:
Doesn't the underlying system error, though, if that command is wrong? Or do we silently swallow errors?

Reply:
Right now we're silently swallowing errors. I'll push a fix for that as well.